jarbas/api/views.py

```diff
@@ -27,8 +27,9 @@ def get(self, request):
         filters = {k: v for k, v in zip(params, values) if v}
 
         # filter suspicions
-        if self._bool_param('suspicions'):
-            self.queryset = self.queryset.suspicions()
+        suspicions = self._bool_param('suspicions')
+        if suspicions is not None:
+            self.queryset = self.queryset.suspicions(suspicions)
```

Comment:
@cuducos, for the API, is it? I already did this part.
If there is more, let me know.

Reply:
Damn… my bad! I missed it. Sorry, mate.

Reply:
two identical PR hahahah Yesterday I was looking to one of them.. "But I saw some comment about (insert here some part), where is it?? 😖"
I will fix the rest today, many thanks, @cuducos